packages/devtools-core/src/App/UserAppContainer/index.tsx

```diff
@@ -99,7 +101,12 @@ const UserAppContainer = (props: Props) => {
             onLoad={handleLoad}
           />
         ) : (
-          <StyledIFrame ref={iFrameRef} className="iframe" data-testid="iframe">
+          <StyledIFrame
+            ref={iFrameRef}
+            className="iframe"
+            data-testid="iframe"
+            onLoad={handleLoad}
```

Comment:
Ok, so I've tested this a bit more. If I remove only this line (onLoad={handleLoad}), the app no longer loads in Firefox, but it works for Chrome. If I remove both this and the above manual call to handleLoad(), it works in Firefox, but not in Chrome. So we do need both of these

The one thing we do not need for the initial load is the eventListener for "load" event, so I'll remove that bit in the next commit.